ironfish-cli/src/commands/chain/asset.ts

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { IronfishCommand } from '../../command'
```

Comment:
This seems like a separate change. We can remove this and keep it in a separate PR

Reply:
Yes these 2 PRs shouldn't be conflated. Split this out into a different PR.